bindings/nodejs/tests/types/block.spec.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import 'reflect-metadata';
+
+import { expect, describe, it } from '@jest/globals';
+import * as basic_block_tagged_data_payload_json from '../../../../sdk/tests/types/fixtures/basic_block_tagged_data_payload.json';
+import * as basic_block_transaction_payload_json from '../../../../sdk/tests/types/fixtures/basic_block_transaction_payload.json';
+import * as validation_block_json from '../../../../sdk/tests/types/fixtures/validation_block.json';
+import * as protocol_parameters from '../../../../sdk/tests/types/fixtures/protocol_parameters.json';
+import { BasicBlockBody, Utils, ProtocolParameters, Block, HexEncodedString, TaggedDataPayload, Transaction, PayloadType, SignedTransactionPayload, ValidationBlockBody } from '../../';
+
+describe('Block tests', () => {
+
+    it('compares basic block tagged data payload from a fixture', async () => {
+        const block = basic_block_tagged_data_payload_json.block as unknown as Block;
+        expect(block.isBasic).toBe(true);
+        expect(block.body.isBasic).toBe(true);
+        expect(block.asBasic).toBeInstanceOf(BasicBlockBody);
+
+        const basic_block = block.body.asBasic();
+        expect(basic_block.payload).toBeInstanceOf(TaggedDataPayload);
+        const tagged_data_payload = basic_block.payload as unknown as TaggedDataPayload;
+        expect(tagged_data_payload.type).toBe(PayloadType.TaggedData);
+        expect(basic_block.maxBurnedMana).toEqual(864);
```

Comment:
Why do all these expects? Hash would fail if there was an issue

Reply:
yes, but it's less indicative. If we'ld do a change in the type, then the test would just fail with a different hash than expected, but left us no other clue. With the expects it's more likely to get a hint how to fix the issue.

Reply:
But it seems so arbitrary, why `maxBurnedMana` only and not all fields ?

Reply:
good point ... I took it from the previous tests (as they had been approved before)

Reply:
okay ... fine with me!

Reply:
Done